nextflow_schema.json

```diff
@@ -442,6 +450,12 @@
                     "type": "boolean",
                     "description": "Specifies whether or not to filter results based on a list of candidate genes specified in 'vep_filters'.",
                     "fa_icon": "fas fa-toggle-on"
+                },
+                "vcf2cytosure_switch": {
+                    "type": "boolean",
+                    "description": "Turn on the vcf2cytosure subworkflow",
+                    "fa_icon": "fas fa-toggle-on",
+                    "help_text": "vcf2cytosure can generate CGH files from a structural variant VCF file that can be analysed in the CytoSure interpretation software. Cut offs for allele frequencies and bin sizes can be modified in the config file. Turned off by default."
```

Comment:
Instead of switch, I would call it something along the lines of skip_vcf2cytosure and have it set to true by default, that way it would be similar to hoe the rest of the switches are build.

Reply:
I know what you mean, it would be great to keep the naming consistent in the pipeline. I had it exactly like you suggested at first but ran into issues with turning turning the sub-workflow on using the command line. Since the flag was set to true by default, giving the option `--skip_vcf2cytosure` will just skip it again. 

I will try to find a solution